components/NewMeetingForm/NewMeetingForm.js

```diff
@@ -143,6 +143,7 @@ export default function NewMeetingForm({
         alt="speichern Icon"
         source="/check-icon-round.svg"
         text="speichern"
+        functionality="safe"
```

Comment:
"safe" or "save"?

Reply:
hehe.. teacher Paul and coach Paul are very right about this one. Will change it asap :D

Reply:
hehe...Coach Paul has eyes like eagle. Even the smallest things can't be hidden. 🤣